docs/source/getting-started/assessing-utility/noise.ipynb

```diff
@@ -0,0 +1,527 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What Noise Should I Choose?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Thus far, we've been using `.noise()` to make our queries differentially private. However, there are different types of noise available including Laplacian noise and Gaussian noise. In this notebook, we will describe how the two noise distributions differ and how you can choose one which is most appropriate for your particular analysis. \n",
+    "\n",
+    "The default `.noise()` method chooses one of these distributions for you based on the limited information you've provided, specifically the privacy measure. If the privacy measure requires pure-DP to be satisfied, then the Laplace distribution will be used for noise. Otherwise, the Gaussian distribution may be used to satisfy zero concentrated DP. However, with a better understanding of both distributions, you can specify the type of noise and its parameters for higher utility!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import polars as pl \n",
+    "import opendp.prelude as dp\n",
+    "import seaborn as sns \n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np \n",
+    "\n",
+    "dp.enable_features(\"contrib\")\n",
+    "sns.set_theme(style='darkgrid')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Laplace vs. Gaussian"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Laplace and Gaussian distributions are the two primary mechanisms to add noise to ensure differential privacy and preserve relationships for statistical analysis. The two distributions may appear similar at first glance, but there are key differences in their use in differential privacy techniques. \n",
```

Comment:
They don't really appear similar, to be honest. They have pretty different tail properties, so can't really say that.